### PR TITLE
Update readme and fix travis bug

### PR DESCRIPTION
**Pull request by guilhermebodin**
@blegat could you help me to list which ones use the primal and which ones use the dual. 

I think after this list we can make a new release.

### CODE REVIEW

**Comment by blegat** [comment]
Dual form aka geometric form (affine in cone) for CDCS SCS ECOS SeDuMi
Primal form aka geometric form (variable in cone) for SDPT3 SDPNAL CSDP SDPA Mosek

**Comment by joaquimg** [comment]
Both are called geometric form?

**Comment by blegat** [comment]
Ups no I corrected. It's maybe better to not use the primal and dual form wording as it's confusing and I don't think there is any clear convention. Standard and geometric form sounds better

**Comment by joaquimg** [comment]
Makes sense!
Maybe we add the equations to be precise.

File: docs/src/manual.md [hunk 1]
```diff
@@ -161,6 +161,38 @@ You can solve a primal problem by using its dual formulation using the `DualOpti
 DualOptimizer
 ```
 
+Solving an optimization problem via its dual representation can be useful because some conic solvers assume the model is in the standard form and others use the geometric form.
+
+Standard form has affine expressions in cones
```
Comment (blegat) [inline docs/src/manual.md]: This is the opposite. Below is the geometric form

Comment (guilhermebodin) [reply docs/src/manual.md]: My bad hehehe

**Pull request merged**